### PR TITLE
evaluate difficulty of each training sample based on their training loss

### PR DESCRIPTION
**Pull request by Emrysil**
and order the training data based on the training loss

### CODE REVIEW

**Comment by Emrysil** [comment]
Steps:
1. Run 10 epochs using bart.base model to get the checkpoint_best.pt. Use 
- source_train: ABC.train.gold.bea19.orig.txt 
- target_train: ABC.train.gold.bea19.corr.txt 
- source_train: ABCN.dev.gold.bea19.orig.txt 
- target_train: ABCN.dev.gold.bea19.corr.txt 
2. Use the eval_difficulty.py file to generate the individual training loss for each sample. Sort the sample by ascending training loss and write the sorted samples into source and target txt files.
3. Partition the sorted data into 3 sets and train the model with each set:
- use bart.base/model.pt to train with the easiest difficulty partition to get checkpoint_best.pt
- use checkpoint_best.py to train with the medium difficulty partition to get checkpoint_best2.pt
- use checkpoint_best2.py to train with the medium difficulty partition to get checkpoint_best3.pt
- You need to update the model path in train.sh to use the latest fine-tuned model version
